src/ui-test/tests/commands.openapi.test.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { WebDriver, SideBarView, VSBrowser, ActivityBar, EditorView, Workbench, InputBox, DefaultTreeSection } from "vscode-uitests-tooling";
+import { RESOURCES, waitUntilExtensionIsActivated, deleteFile, killTerminal, waitUntilEditorIsOpened } from "../utils/testUtils";
+import * as pjson from '../../../package.json';
+import { expect } from "chai";
+
+let driver: WebDriver;
+let input: InputBox;
+let sideBar: SideBarView;
+
+describe('Create a Camel Project using command', function () {
```

Comment:
label of this `describe` is not correct I would suggest to after change also modify the `it` label, eg. could be sufficient

```typescript
describe('Create a Camel Route using command', function () {
    it('OpenAPI', async function () {
```